app/models/piece.rb

```diff
@@ -67,4 +68,9 @@ def path_obstructed?(x, y)
     return diagonal_obstructed?(x, y) if diagonal_move?(x, y)
     false
   end
+
+  def opposing_color
```

Comment:
Oops! Nothing yet. I was waiting to find out how tyhna ended up representing the piece color and forgot about it. I'll change it when I comment out the move stuff.